src/openforms/appointments/utils.py

```diff
@@ -82,18 +92,20 @@ def book_appointment_for_submission(submission: Submission) -> None:
         appointment_id = client.create_appointment(
             [product], location, start_at, appointment_client
         )
-        submission.appointment_information = f"Appointment Id: {appointment_id}"
         AppointmentInfo.objects.create(
             status=AppointmentDetailsStatus.success,
             appointment_id=appointment_id,
             submission=submission,
         )
-    except AppointmentCreateFailed as e:
+    except AppointmentCreateFailed:
         AppointmentInfo.objects.create(
             status=AppointmentDetailsStatus.failed,
             error_information="Failed to make appointment",
             submission=submission,
         )
+        raise AppointmentRegistrationFailed(
+            "Unable to create appointment", should_retry=True
```

Comment:
Just to double check, this `should_retry` should be set to `True` because the task itself is what handles the timeout, meaning if this exception is raised after the task's timeout limit then the task won't actually retry, correct?

Looking at the code if this was not specified then it would default to `False` and the task would not retry even within the timeout limit.

Reply:
yeah, this seems to be correct. There would be no retry if there's insufficient (=only partial) data submitted by the end-user, but if it's because JCC/Q-matic errored out, then we need to retry.